moviepy/audio/fx/audio_loop.py

```diff
@@ -17,12 +17,17 @@ def audio_loop(audioclip, nloops=None, duration=None):
     >>> videoclip.set_audio(audio)
 
     """
+    try:
```

Comment:
I did this to avoid exclude it during the `VideoClip` patching

Reply:
Isn't this the same as using the `apply_to_audio()` decorator?

Reply:
Oh there's actually an `audio_video_fx()` decorator that the other audio fx use. Is there a reason that it cannot be applied here?